lib/knapsack_pro/config/env.rb

```diff
@@ -200,6 +200,15 @@ def rspec_test_example_detector_prefix
           ENV.fetch('KNAPSACK_PRO_RSPEC_TEST_EXAMPLE_DETECTOR_PREFIX', 'bundle exec')
         end
 
+        def slow_test_file_threshold
+          value = ENV.fetch('KNAPSACK_PRO_SLOW_TEST_FILE_THRESHOLD', nil)
+          value.to_f unless value.nil?
```

Comment:
It seems very unobvious to me that this line's return value would be `nil` if `value` is `nil`. Yet, we are relying on the strict return value of the method on L209.

I propose to make one (or both) of the below changes:

L205:
```ruby
value&.to_f
```
L208:
```ruby
!!slow_test_file_threshold
```